lib/new-config-manager.js

```diff
@@ -162,6 +162,11 @@ const getCryptoUnits = (crypto, config) =>
 
 const setTermsConditions = toNamespace(namespaces.TERMS_CONDITIONS)
 
+const getCashOutConstraints = (cryptoCode, settings) => {
+  const CASH_IN_ONLY_WALLETS = ['coinbasepro']
+    return _.includes(getWalletSettings(cryptoCode, settings).wallet, CASH_IN_ONLY_WALLETS)
+}
+
```

Comment:
There's a different identation on the return for no specific reason

```suggestion
const getCashOutConstraints = (cryptoCode, settings) => {
  const CASH_IN_ONLY_WALLETS = ['coinbasepro']
  return _.includes(getWalletSettings(cryptoCode, settings).wallet, CASH_IN_ONLY_WALLETS)
}

```